content/docs/coldchain/equipment.md

```diff
@@ -82,34 +82,68 @@ template which can be used to import assets.
+Assets are automatically assigned to the logged in store.
 
-<!--
-TODO: Add link to central server docs when they are created
--->
-
-When importing assets on the central server, an additional field of `store` will be required which will assign assets to their respective stores.
+This differs when importing assets on the Open mSupply Central Server - an additional field of `store` will be required to assign assets to their respective stores.
 
 Information will need to be entered in the format provided by the csv template in order for Open mSupply to be able to process and upload this data.
 
-| Column                  | Description                                                      | Example            | Mandatory         |
-| :---------------------- | :--------------------------------------------------------------- | :----------------- | :---------------- |
-| **Asset Number**        | The asset number assigned by the user's organisation             | 1                  | \*                |
-| **Catalogue item code** | The Asset Catalogue code - i.e. the kind of asset to be imported | E003/002           | \*                |
-| **Store**               | The code of the Store to which the asset is to be assigned       | CMS                | \* (central only) |
-| **Notes**               | General notes about the asset                                    | Kept in EPI office |                   |
-| **Serial Number**       | The serial number of that particular asset                       | 123456789ABC       |                   |
-| **Installation date**   | The date that the asset was installed at the Store               | 2024-01-14         |                   |
-
-A csv file can be uploaded once it has been created in the example format.
+| Column                  | Description                                                                                | Example                    | Mandatory         |
+| :---------------------- | :----------------------------------------------------------------------------------------- | :------------------------- | :---------------- |
+| **Store**               | The code of the Store to which the asset is to be assigned                                 | CMS                        | \* (central only) |
+| **Asset Number**        | The asset number assigned by the user's organisation                                       | 1                          | \*                |
+| **Catalogue item code** | The [Asset Catalogue code](/docs/catalogue/assets) - i.e. the kind of asset to be imported | E003/002                   | \*                |
+| **Installation date**   | The date that the asset was installed at the Store                                         | 2024-01-14                 |                   |
+| **Replacement due**     | The date that the asset is due for replacement                                             | 2026-01-14                 |                   |
```

Comment:
Looks like this is now DD/MM/YYYY? After Noels changes.

Also it's not consistent in the UI anymore :(

![image](https://github.com/user-attachments/assets/144dcc26-cc80-425d-88c0-75d7d44c62cc)


Personally I'm not convinced on the change to DD//MM/YYYY for imports, I like the old format better...

Reply:
Ugh yeah, we made about 4 PRs into this modal at the same time, I think it needs one more. I don't really mind the date format either way, but if one maps better to/from the exports, I'd be inclined to go with that. 

Probably not high priority now 😢 Will update here to be consistent, thanks for catching 😅